lib/folio/engine.rb

```diff
@@ -124,7 +124,7 @@ class Engine < ::Rails::Engine
             deprecations << "Missing index_folio_pages_on_by_query index on folio_pages. That is probably caused by using traco title_* attributes. Add a custom one."
           end
 
-          if !Rails.env.test? && ActiveRecord::Base.connection.exec_query("SELECT id FROM folio_email_templates LIMIT 1;").rows.size == 0
+          if !Rails.env.test? && ActiveRecord::Base.connection.exec_query("SELECT id FROM folio_email_templates WHERE mailer ='Folio::DeviseMailer' LIMIT 1;").rows.size == 0
```

Comment:
@foton Proc tohle? Jinak :+1:

Reply:
Protože když jsem už měl nahrané v DB nějaké  Mail templates z Auctify ( `Auctify::ApplicationMailer`) tak mi to nenhrálo ty z Folia. Takhle je to nahraje.

Reply:
@mreq ^